.circleci/config.yml

```
@@ -338,8 +338,8 @@ jobs:
           command: bundle exec fastlane ios ci-run | tee ./logs/build
           environment:
             GIT_COMMIT_DESC: $(git log --format=oneline -n 1 $CIRCLE_SHA1)
-      - run: yarn detox build --configuration ios.sim.release
-      - run: yarn detox test --configuration ios.sim.release --cleanup
+      - run: yarn detox build --configuration ios.sim.release | xcpretty
+      - run: yarn detox test --configuration ios.sim.release --cleanup | xcpretty
```

Let's just not pipe test output through `xcpretty`

```suggestion
      - run: yarn detox test --configuration ios.sim.release --cleanup
```

---

Why, though? I still don't understand how xcpretty, which is only slurping stuff that looks like `xcodebuild` output, would possibly slurp jest output.  Expect the Proof of Concept that it actually does report failures shortly, but I see no reason to make an exception and keep the noise in the test output, especially if that's wholly unnecessary.

---

I see no reason to even pipe it through. By that argument, why not pipe literally every command through xcpretty? 

I just don't see any reason to pipe this through.